app/client/keybase/debug/keystore.go

```diff
@@ -5,8 +5,8 @@ package debug
 import (
 	"fmt"
 	"os"
-	"sync"
 
+	"github.com/korovkin/limiter"
```

Comment:
Team, LMK if you'd rather prefer not to add a new dependency and redo this with `sync` package or channels.